introduction_to_amazon_algorithms/xgboost_abalone/xgboost_parquet_input_training.ipynb

```diff
@@ -63,8 +64,8 @@
     "from sklearn.datasets import load_svmlight_file\n",
     "\n",
     "# Download the dataset and load into a pandas dataframe\n",
-    "FILE_DATA = 'abalone'\n",
-    "urllib.request.urlretrieve(\"https://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data\", FILE_DATA)\n",
+    "FILE_NAME = 'abalone.csv'\n",
+    "urllib.request.urlretrieve(\"https://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data\", FILE_NAME)\n",
     "feature_names=['Sex', \n",
```

Comment:
Thanks for working on fixing this notebook. The notebook as currently written will still fail even with your fixes:
```
ValueError: could not convert string to float: 'F'
```
or
```
ValueError: could not convert string to float: 'M'
```
This is due to the "Sex" column being a categorical column: ["F", "M"]. When I encoded it with
```
data["Sex"] = data["Sex"].astype("category")
data["Sex"] = data["Sex"].cat.codes
```
the training job succeeded and the notebook worked end-to-end. I know this bug was there before you started working on this notebook, but can we fix it while we are at it?

Reply:
Addressed, thanks Edward